### DISCUSSION TITLE
How can I test a component with <Show>?

### DISCUSSION THREAD

**Discussion by kevinnascimento** [chakra-ui/chakra-ui#7049] [Q&A]
Hi Everyone,

I would like to test a component that I'm using <Show> to show/hide other components just on mobile devices.

```js
 <Show above="md">
    <Box
       w='full'
       color={'gray.200'} >
       Test
    </Box>
 </Show>
```

So, if in my test I'm looking for the word 'Test' it doesn't find it. How can I ensure that my test is above or below 'MD'? 

Any idea how I can do it?

Thank you!!

**Comment by Tisks** [comment]
What kind of testing library are you using? 
Jest, Enzyme, Mocha...?

In those cases I would mock the screen width like:

```
global.innerWidth= 700; //700px
```

and then render the component.

Depending on what `md` means (48em that is 768px by default in chakra-ui or other if you defined it in the theme).

In the example it would show the text 'Test' and if you want to test for it not showing just change the innerWidth to lower than `md`
